stable/ghost/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: ghost
-version: 0.4.15
-appVersion: 0.11.10
+version: 0.4.16
```

Comment:
shouldn't we move this to `0.5.0` at least?
or `1.x.y`?

Reply:
Fair point. @sameersbn?

Reply:
Yes, it should. The bot doesn't have context for this so we need to judge it manually. Thanks for bringing it up @luxas!